mobile/src/app/components/resources/resources.component.html

```diff
@@ -20,7 +23,7 @@
           </div>
         </div>
         <div *ngIf="resource != data[data.length - 1]">
-          <hr class="i-mt-150 w-95" />
+          <!--<hr class="ba b--black-05 mt4" />-->
```

Comment:
Should this be commented out?

Also, I did like your screencap of the Resources w/o the `hr` tag. So if you're in favor of dropping it, I support 👍

Reply:
okay, will be removing then!